mkdocs/structure/files.py

```diff
@@ -67,6 +67,8 @@ def add_files_from_theme(self, env, config):
         """ Retrieve static files from Jinja environment and add to collection. """
         def filter(name):
             patterns = ['.*', '*.py', '*.pyc', '*.html', '*readme*', 'mkdocs_theme.yml']
+            # Exclude translation files
+            patterns.extend(['*.pot', '*.po', '*.mo'])
```

Comment:
I wonder if it makes more sense to exclude the `locales` dir instead: `locales/*`. 

And while I'm thinking of it, these file types need to be added to `MANIFEST.in` so they get included in the package.

Reply:
On the same subject: which files should I leave behind? `.pot` and `.mo` files are ignored by git.

What I did to generate `.mo`:
1. Create a babel config file with content:
    ```
    [jinja2: **.html]
    extensions=jinja2.ext.i18n
    ```
2. `pybabel extract -F locales/babel.cfg -o locales/messages.pot .`
3. `pybabel init -d locales -l en -i locales/messages.pot`
4. `pybabel compile -d locales`

Reply:
The `.pot` and `.mo` files are ignored by git because they are listed in the [.gitignore][1] file. The list of files there is just the standard list for Python projects. Personally I'm not familiar enough with the various translation files types to know which should be ignored and which shouldn't. But given these are in the list of standard file types to ignore, my assumption is that they should be. Presumably they would be generated files? We likely need to add a command (or few) to the tox file to generate the translations before running the tests.

[1]: https://github.com/mkdocs/mkdocs/blob/master/.gitignore#L48-L50

Reply:
That's what I meant by ignored! I'm not that much familiar with translation files either but:

- `.pot` file is the output of 2 and probably can be generated from a `.po` file
- `.po` files are the output of 3 (they are then manually modified and can be updated with `update` command, see [here](http://babel.pocoo.org/en/latest/cmdline.html))
- `.mo` files are the output of 4

I modified the failing test so it doesn't depend on a `.mo` file. Note: `.mo` files would have to be generated before packaging (manually or preferably automatically).

Reply:
Any files which need generated before packaging should also be generated before testing. Perhaps [here][1]?

When I have some more time, I'm going to need to take a closer look at this to figure out how to proceed.

[1]; https://github.com/mkdocs/mkdocs/blob/master/tox.ini#L15